kdump/include/runtest.sh

```diff
@@ -19,8 +19,8 @@
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 
-# Source rhts environment 
-. /usr/bin/rhts-environment.sh
+# Source  environment
+. ../../cki_lib/libcki.sh
```

Comment:
It seems that most other imports of `libcki` have an `|| exit 1` at the end. It would be nice to add that everywhere. Also, use `.` in the (few) places we're using `source`. Or use `source` if you prefer, of course, the point is being consistent.

Reply:
I use .  when adding libcki  
could missed  exit 1 , going to check.

Reply:
Just push additional  commit  to add "exit 1"